src/main/java/bio/terra/tanagra/query/SqlPlatform.java

```diff
@@ -0,0 +1,7 @@
+package bio.terra.tanagra.query;
+
+public enum SqlPlatform {
+  BIGQUERY,
```

Comment:
Could this just use the same values that `[Data Pointer](https://github.com/DataBiosphere/jade-data-repo/pull/1516/files?diff=unified&w=1#diff-8b72dc7ef0453d3d420185f37e7f114f44e95b274742abeeb1993bd9cffdc829R6)` uses? `BQ_DATASET` and `AZURE_DATASET`? Then we wouldn't need to redefine for DataPointer

Reply:
Actually I think the DataPointer ones will go away, they're only used when creating the query executor classes from serialization (which is not part of this PR). When query execution is added back in, TDR knows if a dataset is GCP or azure so it wouldn't need to be part of DataPointer.